frontend/src/pages/ActivityReport/constants.js

```diff
@@ -48,6 +48,10 @@ export const nonGranteeParticipants = [
 ];
 
 export const targetPopulations = [
+  'Infants and Toddlers (ages birth to 3)',
+  'Preschool (ages 3-5)',
+  'Pregnant Women',
+  '--------------------',
```

Comment:
Would double check if they want this -------------------- in there or if it was just in the Jira as a seperator. Can't imagine we want people picking that.

Reply:
Right now its not selectable. But I will double check.